### DISCUSSION TITLE
Continuous Project?

### DISCUSSION THREAD

**Discussion by pipd0un** [salah-rashad/blueprint_system#3] [General]
Do you plan to continue this awesome project? Exporting the blueprint as json and having the connections of the nodes to each other, saving features would be very nice. I am working on a project and if these features will be available in the future, I will continue to follow.

**Comment by salah-rashad** [comment]
Yes I will.

Thank you for your feedback, I realized that I need to take a break learning new things and techniques to improve my coding skills with a clean code.

I think it's time to continue working on this package. 🙏
